discovery-api/values.yaml

```diff
@@ -1,24 +1,47 @@
-replicaCount: 3
+global:
+  buckets:
+    region: us-west-2
+    hostedFileBucket: "dataset-bucket"
+  kafka:
+    brokers: pipeline-kafka-bootstrap:9092
+  presto:
+    url: http://platform-kubernetes-data-platform-presto:8080
+  redis:
+    host: redis.external-services
+    password: ""
+  vault:
+    endpoint: vault.vault:8200
+  ingress:
```

Comment:
Are all these ingress values still used with the annotation changes?

Reply:
So I don't believe so, but they were being populated with what looked like actual annotations, so we put them here for documentation but I am now reconsidering having scheme, subnets and security_groups here

Reply:
If the values are no longer being referenced then they need to be removed

Reply:
Removed